### PR TITLE
build windows 2025 binaries

### PR DESCRIPTION
**Pull request by paulyufan2**
**Reason for Change**:

CNS and CNI windows 2025 binaries can be built locally

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] relevant PR labels added

**Notes**:

### CODE REVIEW

**Review by jpayne3506 — APPROVED** [review]

**Review by camrynl — APPROVED** [review]

**Review by nddq — APPROVED** [review]

File: cns/Dockerfile [hunk 1]
```diff
@@ -28,6 +28,9 @@ FROM mcr.microsoft.com/windows/servercore@sha256:6fdf140282a2f809dae9b13fe441635
 # intermediate for win-ltsc2022
 FROM mcr.microsoft.com/windows/servercore@sha256:45952938708fbde6ec0b5b94de68bcdec3f8c838be018536b1e9e5bd95e6b943 as ltsc2022
 
+# intermediate for win2025
+FROM mcr.microsoft.com/windows/servercore:ltsc2022-KB5035857-amd64 as win2025
```
Comment (nddq) [inline cns/Dockerfile:32]: can we use a sha digest here instead of a tag?

Comment (paulyufan2) [reply cns/Dockerfile:32]: it's we do not have official ws2025 image and we use this temporarily to build win2025 binaries/docker images, so can we will use sha in the next release

**Pull request merged**